README.md

```diff
@@ -1,7 +1,102 @@
+* **Product Backlog**: triaged issues that are not part of the current sprint or an upcoming sprint.
+* **Next Sprint**: issues that have been labelled and are being groomed for an upcoming sprint. Note: all upcoming sprints (there can be several being planned in parallel) share this pipeline.
+* **Current Sprint Backlog**: issues that are part of the current sprint that are not being worked on yet.
+* **In Progress**: issues that are being worked on.
+* **Done**: White shores, and beyond, a far green country under a swift sunrise.
+
+### Starting the sprint
+At the start of the sprint, the _Scrum master_ will update the [primary Zenhub workspace](https://app.zenhub.com/workspaces/taskcluster-5ed15d37c2d9744af28567dc/board) to mark the current milestone as active and moved the issues associated with that milestone into the Current Sprint Backlog pipeline. We will hold a kick-off meeting where the _Product owner_ will provide context enough for people to get started and assist developers in choosing their first issue to work on. The _Scrum master_ will remind everyone of any process changes being adopted for the new sprint. Work begins.
+
+### During the sprint
+The team will hold daily stand-up meetings for the duration of the sprint. Our daily stand-ups take place in Slack at 8am PT (11am ET/5pm CEST), except on Tuesdays when we present our stand-up status as the first agenda item of our weekly team meeting. Whether delivered in person on via Slack, the format of the update is the same:
+* DONE: work that has been completed
+* TODO: work that is coming up next
+* BLOCKERS: what is preventing you from getting the work done
+* INTERRUPTS: what is delaying you from getting the work done
+
+The _Product owner_ is paying attention to these updates to make sure that the most important things are being worked on and are being picked up next.
+
+The _Scrum master_ is paying attention to these updates to make sure that blockers are highlighted and interrupts are kept to a minimum.
+
+Once a week, the _Product owner_ and the _Scrum master_ will meet to assess sprint progress. Using the burndown chart, they figure out whether the sprint is still on track and can be completed within the allotted time. If necessary, the critical path can be refined to allow partial delivery of the original objective. Under extraordinary circumstances, they can opt to end the current sprint.
+
+### Ending the sprint
+As the sprint draws to a close, the _Product owner_ and _Scrum master_ will meet to decide what issues from the _Current Sprint Backlog_ (if anything) still needs to be done. The _Scrum master_ will move all issues that have not been started to the Product Backlog pipeline. The _Product owner_ will engage with developers who still have work in progress to figure out how and when to deploy that wrap-up that work during the _Time between sprints_.
```

Comment:
Will the meeting with you (scrum master) and the the product owner happen during their weekly 1:1 or would this be an additional meeting in which the day and time should be decided during the first week of the sprint?

Reply:
> Will the meeting with you (scrum master) and the the product owner happen during their weekly 1:1 or would this be an additional meeting in which the day and time should be decided during the first week of the sprint?

Where it makes sense, yes. There may be some future sprint where I'm not acting as scrum master and there may not be an existing 1x1 to leverage, but I'll call it out explicitly in the docs for now. Thanks!